src/planner/expand_hypertable.c

```diff
@@ -1538,7 +1538,12 @@ ts_plan_expand_hypertable_chunks(Hypertable *ht, PlannerInfo *root, RelOptInfo *
 
 		/* if we're performing partitionwise aggregation, we must populate part_rels */
 		if (rel->part_rels != NULL)
+		{
 			rel->part_rels[i] = child_rel;
+#if PG15_GE
+			rel->live_parts = bms_add_member(rel->live_parts, i);
+#endif
+		}
```

Comment:
This part looks good, but I do not understand why you do not need to set this in `data_node_scan_plan.c` as well?

Reply:
Bringing it back since it passes more tests this way.

Reply:
Then it would be good to understand why.